src/services/hsm.service.js

```diff
@@ -219,17 +219,33 @@ class HsmSession {
       });
     }
 
-    /* istanbul ignore next */
-    if (tx.version === hathorLib.constants.NANO_CONTRACTS_VERSION) {
+    /** @type {hathorLib.Address} */
+    let address = null;
```

Comment:
Same changes as the fireblocks file, due to how we changed the signature for nano contracts on the lib